### ISSUE TITLE
Support disabling auto_complete on job handler completion

### ISSUE THREAD

**Issue by PSanetra** [camunda-community-hub/pyzeebe#528]
[The spring-zeebe-sdk supports a `autoComplete` parameter on JobWorker annotations.](https://docs.camunda.io/docs/apis-tools/spring-zeebe-sdk/configuration/#auto-completing-jobs) It would be nice if pyzeebe could support a similar setting for job handlers. A use case would be the creation of a long running asynchronous background task on job activation. The job might be even marked as completed by a different application instance.

**Comment by hoodee86** [comment]
Absolutely, this parameter is so important and useful in some complex scenarios.
